packages/components/Menu/src/Menu/Menu.types.ts

```diff
@@ -23,6 +23,10 @@ export interface MenuProps extends MenuListProps {
    */
   open?: boolean;
 
+  /**
+   *  Call back when the component request value change via controlled state variable
+   */
+  setControlledOpen?: React.Dispatch<React.SetStateAction<boolean>>;
```

Comment:
This information is already sent up by the onOpenChange callback. Why does this need to be added?

Reply:
Removed, used onOpenChange.